mappings/net/minecraft/server/world/SleepManager.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5838 net/minecraft/server/world/SleepManager
+	FIELD field_28866 total I
+	FIELD field_28867 sleeping I
+	METHOD method_33811 clearSleeping ()V
+	METHOD method_33812 canSkipNight (I)Z
+		ARG 1 percentage
+	METHOD method_33813 isLongEnough (ILjava/util/List;)Z
+		ARG 1 percentage
+		ARG 2 players
+	METHOD method_33814 update (Ljava/util/List;)Z
+		COMMENT Updates the sleeping players.
+		COMMENT
+		COMMENT @return {@code true} if the sleeping players or total players have
+		COMMENT changed
+		ARG 1 players
+	METHOD method_33815 countSleeping ()I
```

Comment:
This just returns `sleeping`:

```suggestion
	METHOD method_33815 getSleeping ()I
```

Reply:
sounds weird... it's like a mix of boolean and getter

Reply:
Updated to `getSleeping`.